app/views/api/release_notes/release_notes.yml

```diff
@@ -21,6 +21,37 @@
 #
 #     **bold** and _italic_ text
 
+- title: Updated validation rules for participant action endpoints
```

Comment:
Following the other release notes, I think we should still be clear that these endpoints are now released on the Register ECTs API, as they haven't been previously. https://sandbox.register-early-career-teachers.education.gov.uk/api/guidance/release-notes